sigmet/au3_functions.py

```diff
@@ -141,17 +152,17 @@ def ARIMA_predictor(series, start_date, params=(5, 1, 1)):
 
         # Initialize model
         model = ARIMA(before, order=params)
-        
+
         # Fit the model
         model_fit = model.fit(disp=0)
 
         # Return the forecast as a pd.Series object
         return pd.Series(model_fit.forecast(steps)[0])
     except ValueError:
-        raise ValueError("Cannot provide an SARIMAX forecast for given trend")
+        raise ValueError("Cannot provide an ARIMA forecast for given trend")
 
 
-def SARIMAX_predictor(series, start_date, params=(5, 1, 1)):
```

Comment:
Will this be ARIMA or SARIMAX?

Reply:
SARIMAX for now, but it'll be configurable once we build that functionality